src/Common/tests/System/Net/Configuration.Certificates.cs

```diff
@@ -29,8 +31,18 @@ public static partial class Certificates
 
             private static X509Certificate2Collection GetCertificateCollection(string certificateFileName)
             {
+                // On Windows, .Net Core applications should not import PFX files in parallel to avoid a known system-level race condition.
+                // This bug results in corrupting the X509Certificate2 certificate state.
+                if (!m.WaitOne(MutexTimeout))
+                {
+                    throw new InvalidOperationException("Cannot acquire the global certificate mutex.");
```

Comment:
You could also throw an Xunit.Sdk.XunitException. See: https://github.com/dotnet/corefx/blob/master/src/Common/tests/System/AssertExtensions.cs#L96

That will mark the tests as failed and in that way you will not see an Assert.True fail with expected: True actual: False. Could make it easier to debug.